v1/formats/nswpdc7and8.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name>NSW Premier's Debating Challenge (7 and 8)</name>
+  <short-name>NSWPDC7and8</short-name>
+  <version>1</version>
+  <info>
+    <region>NSW, Australia</region>
+    <level>Primary School</level>
+    <used-at>NSW Premier's Debating Challenge (for Years 7 and 8)</used-at>
```

Comment:
```suggestion
    <level>Years 7 and 8</level>
    <used-at>NSW Premier's Debating Challenge</used-at>
```
(these are just arbitrary strings that aren't used for anything other than display, but this seems more descriptive to me?)